integrationTests/src/test/java/org/sonarqube/gradle/AbstractGradleIT.java

```diff
@@ -103,7 +103,8 @@ protected RunResult runGradlewSonarQubeWithEnvQuietly(String project, String exe
 
   protected RunResult runGradlewWithEnvQuietly(String project, String exeRelativePath, Map<String, String> env, String... args) throws Exception {
     File projectBaseDir = new File(this.getClass().getResource(project).toURI());
-    File tempProjectDir = temp.newFolder(project);
+    String projectDir = project.startsWith("/") ? "." + project : project;
```

Comment:
Tests fail with message `folder path '/java-gradle-simple' is not a relative path` prefixing the `project` with a dot resolves this.